### PR TITLE
feat(icons): added `tickets-plane` icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Concept 

- [ ] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [ ] I've based them on the following Lucide icons: 
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [ ] I've made sure that the icons look sharp on low DPI displays.
- [ ] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [ ] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/plane-tickets.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jguddas"
+  ],
+  "tags": [
+    "plane",
+    "trip",
+    "airplane"
```
Comment (ericfennis) [inline icons/plane-tickets.json:9]: Some extra tags to consider:
```suggestion
    "airplane"
    "flight",
    "travel",
    "fly",
    "takeoff",
    "vacation",
    "passenger",
    "pass",
    "check-in",
    "airport"
```

**Review by ericfennis — COMMENTED** [review]
Nice love the icon, clean and simple!

**Review by ericfennis — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
I think this should be `tickets-plane`, both because it's a ticket variant with a plane on it and because the ticket is larger than the plane itself.

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]
I think this should be `tickets-plane`, both because it's a ticket variant with a plane on it and because the ticket is larger than the plane itself.

**Comment by jguddas** [comment]
> I think this should be `tickets-plane`, both because it's a ticket variant with a plane on it and because the ticket is larger than the plane itself.

100% agree.

**Comment by jguddas** [comment]
> Maybe an idea to add `plane-tickets` as an alias? I can imagine that for some people it can be confusing when used in code.

I'm not a fan of having aliases if we can avoid them.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**